doc/tech/README.md

```diff
@@ -3,203 +3,170 @@
 
-Direct Assignment is supported in x86 architectures through an IOMMU (Input/Ouput Memory Management Unit), which provides the ability for a PCIe device to autonomously (i.e. without hypervisor intervention) perform DMA transfers directly into guest memory as shown in **Figure 5**.
+- Configuration automation over customization
+  - By abstracting the infrastructure capabilities as much as possible, operators are able to use common infrastructure platforms across all VNF vendors.
+  - Maintaining a consistent infrastructure allows for higher levels of automation due to a reduced need for customization of the various components.
+  - Overall, the intention is to reduce the total cost of ownership for operators and development costs for vendors
 
-Once an IO device is directly assigned to a workload, that workload will then have an exclusive access to that device and no other entities (including the hypervisor) can access it.
+- Onboarding and certification
+  - By defining abstracted infrastructure capabilities, and the metrics by which they are measured, the onboarding and certification process for both NFVI and VNFs can be standardized, reducing development time for the VNF developers and deployment and operational management costs for the operators standing up the NFVI environments.
+  - Supply chain, procurement and assurance teams can then use these metrics to more accurately assess the most efficient / best value vendor for a given environment and network services requirement.
 
-This method provides better performance than the para-virtualized one as no hypervisor is involved but provides less flexibility and less portability.
+- Better utilization
+  - Properly mapping VNFs to flavours to the underlying infrastructure, brings the potential for more efficient utilization, than needing to create specific configurations for each type of application in the infrastructure.  
 
-Having an IO device directly assigned to a workload means that the workload needs to run vendor specific drivers and libraries to be able to access that device which makes the workload less portable and dependent on a specific hardware type from a specific vendor which is not aligned with the overall strategy and goals of CNTT and hence this method of IO Virtualization must not be used unless explicitly allowed as an exception as part of the transitional plan adopted by CNTT.
+In conclusion, to serve the stated objective building a common NFVI infrastructure that is able to take advantage of true cloud models for the more rapid development and deployment of SDN NFV applications, the Common Telco NFVI is documentation of a reference model, a select set of architectures and a set of validation and testing suites, so that there is a more consistent model infrastructure for developers and vendors of SDN software and applications to build to.
```

Comment:
Given the discussion in #1168 can I suggest changing `common NFVI infrastructure` to `common cloud infrastructure`.
Also suggest changing `Common Telco NFVI` to `CNTT` as Common Telco NFVI doesn't really make sense in terms of the initialism.

Reply:
I would propose to governance to make CNTT stand for **Common Network-cloud-infrastructure Telco Taskforce**.